ssz-rs/src/ser.rs

```diff
@@ -71,14 +78,15 @@ pub fn serialize_composite_from_components(
 
     // SAFETY: `fixed_lengths_sum` fits in `u32` if the total size check holds
     let mut running_length = fixed_lengths_sum as u32;
-    debug_assert_eq!(fixed.len(), variable_lengths.len());
```

Comment:
This seems to be a duplicate check of the check in line 65, and the variables don't change, so I removed this check.

Reply:
ah thank you! yeah this seems like copypasta slipped in